### PR TITLE
Update accepted affiliated packages

### PR DESCRIPTION
**Pull request by astrofrog**
I know we are going to overhaul this page soon, but just to make sure accepted packages are indeed marked as accepted

### CODE REVIEW

File: affiliated/registry.json [hunk 1]
```diff
@@ -287,6 +287,26 @@
             "home_url": "https://marxs.readthedocs.io/",
             "pypi_name": "marxs",
             "description": "Multi-Architecture-Raytrace-Xraymission-Simulator"
+          },
+          {
+             "name": "regions",
+             "maintainer": "Christoph Deil <Deil.Christoph@gmail.com>, Johannes King <johannes.king@mpi-hd.mpg.de>, and Larry Bradley <lbradley@stsci.edu>",
```
Comment (astrofrog) [inline affiliated/registry.json]: @deil @joleroi @larrybradley - I wasn't sure who to list as primary maintainers. What did we agree previously?

Comment (larrybradley) [reply affiliated/registry.json]: I don't recall a discussion about this, but I'm happy to be included.

Comment (joleroi) [reply affiliated/registry.json]: @adonath should be mentioned unless he doesn't want to

Comment (adonath) [reply affiliated/registry.json]: Thanks, @joleroi! But I think it's totally correct to **not** list me as a maintainer and neither do I want to be listed. My contributions were on the "consulting" side, I think I didn't even wrote a single line of code...

Comment (astrofrog) [reply affiliated/registry.json]: Ok, thanks for clarifying! And @cdeil should definitely be included so I think this is good to go

**Pull request merged**